protocol/contracts/lpp/src/contract/mod.rs

```diff
@@ -173,7 +173,7 @@ pub fn query(deps: Deps<'_>, env: Env, msg: QueryMsg<LpnCurrencies>) -> Result<B
                         .map_err(ContractError::InvalidOracleBaseCurrency)
                         .and_then(|oracle_ref| to_stable(oracle_ref, total, deps.querier))
                 })
-                .map(CoinDTO::<LpnCurrencies>::from)
+                .map(CoinDTO::<PaymentGroup>::from)
```

Comment:
Just to make sure, this didn't leak from another branch, did it?

Reply:
No, this bug did not manifest itself in long protocols since the stable currency in long protocols belongs to Lpn currencies. In the common case, the stable currency belongs to the PaymentGroup.